.github/workflows/updater-check.yaml

```diff
@@ -0,0 +1,61 @@
+name: Start Updater
```

Comment:
@tommartensen if this file looks good, I will delete updater-support-versions.py and start-updater.yaml

Reply:
Looks good! Some minor comments

Reply:
This should be a more specific name. How about `Vulnerability Updater`? Same idea with the file name. I think `updater-check` is a bit vague